### PR TITLE
feat: support eval exports for session

### PR DESCRIPTION
**Pull request by axiomofjoy**
- adds `get_evaluations` method to Phoenix session object

### CODE REVIEW

File: src/phoenix/core/evals.py [hunk 1]
```diff
@@ -171,3 +173,58 @@ def get_document_evaluation_scores(
...
+
+    def export_evaluations(self) -> List[Evaluations]:
+        evaluations: List[Evaluations] = []
+        evaluations.extend(self._export_span_evaluations())
+        evaluations.extend(self._export_document_evaluations())
+        return evaluations
+
+    def _export_span_evaluations(self) -> List[SpanEvaluations]:
+        span_evaluations = []
+        for eval_name, span_evaluations_by_id in self._span_evaluations_by_name.items():
```
Comment (RogerHYang) [inline src/phoenix/core/evals.py:185]: nit. let's make a copy before each iteration
```suggestion
        with self._lock:
            span_evaluations = tuple(self._span_evaluations_by_name.items())
        for eval_name, span_evaluations_by_id in span_evaluations:
```

File: src/phoenix/core/evals.py [hunk 2]
```diff
@@ -171,3 +173,58 @@ def get_document_evaluation_scores(
...
+        return span_evaluations
+
+    def _export_document_evaluations(self) -> List[DocumentEvaluations]:
+        evaluations = []
+        for eval_name, document_evaluations_by_id in self._document_evaluations_by_name.items():
+            span_ids = []
+            document_positions = []
+            rows = []
+            for span_id, document_evaluations_by_position in document_evaluations_by_id.items():
+                for document_position, pb_eval in document_evaluations_by_position.items():
```
Comment (RogerHYang) [inline src/phoenix/core/evals.py:210]: nit. let's apply `sorted` to `document_position`
```suggestion
                with self._lock:
                    pb_evals = sorted(document_evaluations_by_position.items())
                for document_position, pb_eval in pb_evals:
```

**Review by RogerHYang — APPROVED** [review]

File: src/phoenix/core/evals.py (hunk 2 above)
Comment (axiomofjoy) [reply src/phoenix/core/evals.py:210]: Good call

File: src/phoenix/core/evals.py [hunk 3]
```diff
@@ -171,3 +173,58 @@ def get_document_evaluation_scores(
...
+            rows = []
+            for span_id, pb_eval in span_evaluations_by_id.items():
+                result = pb_eval.result
+                row = {}
+                if result.HasField("score"):
+                    row["score"] = result.score.value
+                if result.HasField("label"):
+                    row["label"] = result.label.value
+                if result.HasField("explanation"):
+                    row["explanation"] = result.explanation.value
```
Comment (RogerHYang) [inline src/phoenix/core/evals.py:196]: does this work?
```suggestion
                row = MessageToDict(pb_eval.result)
```

Comment (axiomofjoy) [reply src/phoenix/core/evals.py]: Good suggestion! It worked. Addressed in f0119440.

File: src/phoenix/core/evals.py (hunk 2 above)
Comment (axiomofjoy) [reply src/phoenix/core/evals.py]: addressed in [8be7d5e](https://github.com/Arize-ai/phoenix/pull/2094/commits/8be7d5e9c55527ffa3a1c2e9b6ebf3345209dc1e)

File: src/phoenix/core/evals.py (hunk 1 above)
Comment (axiomofjoy) [reply src/phoenix/core/evals.py]: addressed in [8be7d5e](https://github.com/Arize-ai/phoenix/pull/2094/commits/8be7d5e9c55527ffa3a1c2e9b6ebf3345209dc1e)

**Pull request merged**